Gemfile

```diff
@@ -53,7 +54,7 @@ group :test, :development do
   gem 'pry-rails'
   gem 'pry-byebug'
   gem 'rspec-rails'
-  gem 'rubocop'
+  gem 'rubocop', '0.31.0'
```

Comment:
@paulanthonywilson according to the PR title this PR is to upgrade Nokogiri `Upgrade Nokogiri and other packages to remove security vulnerabilities #418`. Am I right in assuming that this involves everything reported by GitHub on this repo as you've described in the description for this PR?

- [x] Rails > 4.2.5.2
- [x] JQuery-Rails > 4.0.3
- [x] Nokogiri > 1.6.7.2
- [ ] Rubocop > 0.31.0
- [x] RubyZip > 1.2.0

_Rubocop_ has been pinned to `0.31.0` in `Gemfile` as `gem 'rubocop', '0.31.0'`. Shouldn't this version of Rubocop be upgraded too?

Reply:
@Tr4pSt3R  I totally forgot that Rubocop was one of the vulnerabilities, so yes - good point.

Nether-the-less, I think it would make sense to upgrade Rubocop in another PR. Upgrading introduces a lot of rules and subsequent violations, so I'd rather have a go at fixing those in its own PR.

(Also the vulnerability is slight, and rubocop is only included in dev and test.)

Reply:
@paulanthonywilson yeah sure. We can make a separate PR to deal with Rubocop